WeakAuras/Transmission.lua

```diff
@@ -458,12 +469,12 @@ function StringToTable(inString, fromChat)
   return deserialized;
 end
 
-function WeakAuras.DisplayToString(id, forChat)
+function WeakAuras.DisplayToString(id, forChat, unique)
   local data = WeakAuras.GetData(id);
   if(data) then
     local transmitData = CompressDisplay(data);
-    transmitData.controlledChildren = nil;
     local children = data.controlledChildren;
+    transmitData.uid = unique and (transmitData.uid or GenerateUniqueID()) or nil
```

Comment:
You actually need to set the uid in data, so that on next export the uid is the same.